### ISSUE TITLE
Adding annotations to Associations from ClinVar

### ISSUE THREAD

**Issue by adgirish** [bigdatagenomics/gnocchi#28]
Given an RDD of Association objects, trying to add annotations from the ClinVar database providing biological context for these variations.

Annotations will include information about how variant manifests in patients, surrounding genome info, et cetera. 

See [this Google Doc](https://docs.google.com/document/d/1oR6xYGICHRJe46pyFfXPkxjxGTvq-CWBI8e-Sd-Cchk/edit) for more information

**To Do**
- [ ] Translate XML to Parquet > HDFS
- [ ] Generate actual rsID test data
- [ ] Finalize annotation fields and build class

**Comment by adgirish** [comment]
Given an RDD of Association objects, trying to add annotations from the ClinVar database providing biological context for these variations.

Annotations will include information about how variant manifests in patients, surrounding genome info, et cetera. 

See[this Google Doc](https://docs.google.com/document/d/1oR6xYGICHRJe46pyFfXPkxjxGTvq-CWBI8e-Sd-Cchk/edit) for more information

**Comment by fnothaft** [comment]
CC @heuermh most of the functionality for pulling in these annotations should already exist in ADAM's variant annotation codepath.

**Comment by heuermh** [comment]
Right, all the ClinVar annotations in a VCF file will end up in the `VariantAnnotation.attributes` map.
